src/screens/cars/AddCar.tsx

```diff
@@ -0,0 +1,374 @@
+      firstName: currentUserToken?.firstName,
+      name: currentUserToken?.name,
+      status: UserStatus.ACTIVE,
+      role: currentUserToken.role,
+      email: currentUserToken.email
+    } as User;
+    this.setState({
+      selectedUser: this.user ?? currentUser
+    });
+  }
+
+  public componentWillUnmount() {
+    super.componentWillUnmount();
+    Orientation.unlockAllOrientations();
+  }
+
+  public onBack(): boolean {
+    const { navigation } = this.props;
+    const connectorID = Utils.getParamFromNavigation(this.props.route, 'connectorID', null) as unknown as number;
+    const chargingStationID = Utils.getParamFromNavigation(this.props.route, 'chargingStationID', null) as unknown as string;
+    if (connectorID !== null && chargingStationID) {
+      navigation.navigate('ChargingStationsNavigator', {
+        screen: 'ChargingStationConnectorDetailsTabs',
+        params: {
+          params: {
+            chargingStationID,
+            connectorID,
+            user: this.user,
+            tag: Utils.getParamFromNavigation(this.props.route, 'tag', null),
+            showChargingSettings: true
+          }
+        }
+      });
+      return true;
+    } else {
+      navigation.goBack();
+      return true;
+    }
+  }
+
+  public render() {
+    const { navigation } = this.props;
+    const {
+      selectedCarCatalog,
+      selectedConverter,
+      selectedCarCatalogConverters,
+      selectedUser,
+      vin,
+      licensePlate,
+      isDefault,
+      type,
+      addCarPending
+    } = this.state;
+    const commonColors = Utils.getCurrentCommonColor();
+    const modalCommonStyles = computeModalCommonStyle();
+    const formStyles = computeFormStyleSheet();
+    const style = computeStyleSheet();
+    return (
+      <View style={style.container}>
+        <HeaderComponent
+          title={I18n.t('cars.addCarTitle')}
+          navigation={navigation}
+          leftAction={this.onBack.bind(this)}
+          leftActionIcon={'navigate-before'}
+          rightAction={() => {
+            navigation.dispatch(DrawerActions.openDrawer());
+            return true;
+          }}
+          rightActionIcon={'menu'}
+        />
+        <ScrollView style={style.scrollview} contentContainerStyle={style.content}>
+          <ModalSelect<CarCatalog>
+            openable={true}
+            renderItem={(carCatalog: CarCatalog) => <CarCatalogComponent shadowed={true} carCatalog={carCatalog} navigation={navigation} />}
+            renderItemPlaceholder={() => this.renderCarCatalogPlaceholder(style)}
+            onItemsSelected={(selectedCarCatalogs: CarCatalog[]) => this.onCarCatalogSelected(selectedCarCatalogs?.[0])}
+            navigation={navigation}
+            selectionMode={ItemSelectionMode.SINGLE}>
+            <CarCatalogs navigation={navigation} />
+          </ModalSelect>
+          <Input
+            disabled={true}
+            InputComponent={() => (
+              <SelectDropdown
```

Comment:
Use the same inout header name for drop down:

<img width="453" alt="2021-10-04_10-38-24" src="https://user-images.githubusercontent.com/25266945/135820665-0cbebca8-84bf-4c2c-80ec-f1d993f5d71a.png">

Reply:
So should I left the select label empty when there is several choices ?

Reply:
You can display the same place holder as if it was an input field (reported in another comment)

Reply:
Better but still few updates:
- Bg of models should be the same as Bg of input fields
- Same for placeholder: same color

<img width="386" alt="2021-10-04_23-39-35" src="https://user-images.githubusercontent.com/25266945/135928996-5e88b5bf-13eb-40fe-b894-1a72d5282767.png">

Reply:
The placeholder has a different color because it is disabled when no model is selected
I will make it clearer

Reply:
So the same background only in theme light ? In dark theme we won't see that it is clickable because the shadows are black

Reply:
> So the same background only in theme light ? In dark theme we won't see that it is clickable because the shadows are black

Align with the input field's theme.